test_runner/src/main/kotlin/ftl/run/NewTestRun.kt

```diff
@@ -18,29 +18,30 @@ import ftl.run.platform.runIosTests
 import kotlinx.coroutines.TimeoutCancellationException
 import kotlinx.coroutines.withTimeoutOrNull
 
-suspend fun newTestRun(args: IArgs) = withTimeoutOrNull(args.parsedTimeout) {
+suspend fun IArgs.newTestRun() = withTimeoutOrNull(parsedTimeout) {
+    val args: IArgs = this@newTestRun
     println(args)
-    val (matrixMap, testShardChunks, ignoredTests) = cancelTestsOnTimeout(args.project) { runTests(args) }
+    val (matrixMap, testShardChunks, ignoredTests) =
+        cancelTestsOnTimeout(project) { runTests() }
```

Comment:
could be single line

Reply:
single line is not fully visible on 14 cal split screen ;P.

Reply:
single line is not fully visible on 14 cal split screen ;P.